pkg/graph/resolvers/trb_request_form_test.go

```diff
@@ -18,6 +18,14 @@ func (s *ResolverSuite) TestCreateTRBRequestForm() {
 	s.NoError(err)
 
 	s.Run("create/update/fetch TRB request forms", func() {
+		// fetch the form
+		fetched, err := GetTRBRequestFormByTRBRequestID(ctx, s.testConfigs.Store, trbRequest.ID)
+		s.NoError(err)
+		s.NotNil(fetched)
+
+		// confirm that the status is correctly marked "in progress"
+		s.True(fetched.Status == models.TRBFormStatusReadyToStart)
```

Comment:
I usually prefer using `.Equal()` over `.True()` when it's a straight equality check; `.Equal()` will display the difference between the expected and actual values if the test fails. (So here, it'd be `s.Equal(models.TRBFormStatusReadyToStart, fetched.Status)`; the parameter order is expected value first, then the actual value)